TwoPage/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <string name="two_page_app_name">Zwei Seiten</string>
```

Comment:
It can work, but in others we have used the English names. As mentioned before, we should follow the same approach for all sample-names